azure-mgmt-containerregistry/src/main/java/com/azure/management/containerregistry/QueuedBuildOperations.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.azure.management.containerregistry;
+//
+//import com.microsoft.azure.PagedList;
+//import com.microsoft.azure.management.apigeneration.Beta;
+//import com.microsoft.azure.management.apigeneration.Fluent;
+//import rx.Completable;
+//import rx.Observable;
+//
+/**
+ * Grouping of container registry queued build actions.
+ */
+public interface QueuedBuildOperations {
```

Comment:
why this one commented out?

Reply:
I'm not sure about it. As I check in commit, it is moved from track1 and the track1 is already commented out, more than 1 year before. https://github.com/Azure/azure-libraries-for-java/blob/dea6587/azure-mgmt-containerregistry/src/main/java/com/microsoft/azure/management/containerregistry/QueuedBuildOperations.java

Reply:
I think it is due to the queue task was removed from swagger.

Reply:
should we remove this interface then?

Reply:
It could be removed. I'll remove all commented out code in track1.

Reply:
done